pkg/kapp/resources/resources.go

```diff
@@ -557,6 +557,8 @@ func IsResourceChangeBlockedErr(err error) bool {
 		return true
 	case strings.Contains(errMsg, "Internal error occurred: failed calling webhook"):
 		return true
+	case strings.Contains(errMsg, "Resource doesn't exists"):
```

Comment:
`isResourceChangeBlockedErr()` seems to exclusively handle cases where the change is blocked by admission web hooks?
Maybe we could have a separate function like `isWaitingForPlaceholderErr()` which handles this specific case?

Reply:
Makes sense. Will do.

Reply:
Done.